test/it/crd-validation/etcd/specEtcd_test.go

```diff
@@ -0,0 +1,180 @@
+package etcd
```

Comment:
camel casing is not used in file names. please correct them

Reply:
@Shreyas-s14 additionally, please rename the directory `crdvalidation` instead. I think there's also [recommendation](https://go.dev/doc/effective_go#package-names) to not use hyphens in the directory/file names for go projects, and only use underscores or no separators at all.